gateway/ln-gateway/src/client.rs

```diff
@@ -71,6 +71,7 @@ impl GatewayClientBuilder {
         let mut client_builder = Client::builder(db);
         client_builder.with_module_inits(registry);
         client_builder.with_primary_module(self.primary_module);
+        client_builder.migrate_database().await?;
```

Comment:
Shouldn't `ClientBuilder` call that already for the ln-gateway's client? :sus:

Reply:
Hmmm... maybe gateway has it's own builder ... which seems odd. Shouldn't be it be using the standard client builder under the hood anyway?

Reply:
It uses the same `ClientBuilder`, we just create that struct ourself instead of using `make_client_builder`. I think there might have been a refactor in the cli that we didn't do in the gateway.

I can try to change this to use `make_client_builder` if that's preferrable

Reply:
Removed, since this is now called in `build_stopped`